### ISSUE TITLE
Hovrly contacting a foreign trading site? (mql5.com)

### ISSUE THREAD

**Issue by jsnoble7** [tarutin/hovrly#36]
Any reason that Little Snitch is reporting Hovrly contacting a random foreign trading site (mql5.com)? Feels iffy...
<img width="1228" alt="SCR-20241231-kamr-2" src="https://github.com/user-attachments/assets/ffd4b699-8bcf-47b3-8546-1e597e784c44" />

**Comment by tarutin** [comment]
@jsnoble7 hi! we use https://www.finteza.com for tracking daily online users.
Documentation for integration: https://www.finteza.com/en/integrations/insert-code#example

**Comment by tarutin** [comment]
code: https://github.com/tarutin/hovrly/blob/master/app/templates/index.html

**Comment by jsnoble7** [comment]
Fair enough - definitely get that you’re just following Finteza’s documentation.

But have you visited mql5.com? Does it not seem odd that a site trying to help you track daily users is asking you to pull js from a site filled with crypto and trading info?

I’m not a dev at all… but that feels shady af. Something to reconsider to maintain trust of users?

Anyway, I’ve just blocked the domain on Little Snitch, but if definitely doesn’t feel right imho.

**Comment by tarutin** [comment]
I google it. This company develop this tool for analytics — https://www.metatrader5.com/en/news/1901
And — https://www.mql5.com/en/search#!keyword=finteza
